README.md

```diff
@@ -173,6 +174,16 @@ Curious to know how big companies are operating their kafka fleet in production?
 - [Bulletproof Apache Kafka® with Fault Tree Analysis](https://www.confluent.io/kafka-summit-ny19/bulletproof-kafka-with-fault-tree-analysis/) - `2019` - :studio_microphone:
 - [Production Ready Kafka on Kubernetes](https://www.confluent.io/kafka-summit-san-francisco-2019/production-ready-kafka-on-kubernetes/) - `2019` - :studio_microphone:
 
+## Michelin
```

Comment:
Could you help to re-arrange the blog in chronological order, with latest blog at the top of the list ?